pyk/src/tests/integration/kllvm/test_prooftrace.py

```diff
@@ -53,12 +53,12 @@ def test_proof_trace(self, hints: bytes) -> None:
         # check that the first event is the rewrite a() => b()
         assert pt.trace[0].is_step_event()
         assert isinstance(pt.trace[0].step_event, prooftrace.LLVMRewriteEvent)
-        assert pt.trace[0].step_event.rule_ordinal == 96
+        assert pt.trace[0].step_event.rule_ordinal == 97
 
         # check that the second event is the rewrite b() => c()
         assert pt.trace[1].is_step_event()
         assert isinstance(pt.trace[1].step_event, prooftrace.LLVMRewriteEvent)
-        assert pt.trace[1].step_event.rule_ordinal == 97
+        assert pt.trace[1].step_event.rule_ordinal == 98
```

Comment:
I have checked the axioms these new ordinals point to and confirmed that they are the same.